doc/subsystems/shell/legacy_shell.rst

```diff
@@ -1,7 +1,7 @@
-.. _shell:
+.. _legacy_shell:
 
-Shell
-######
+Legacy Shell
```

Comment:
I would actually just remove that, intent is to drop legacy shell support in the next release.

Reply:
I just removed legacy shell documentation

Reply:
@nashif  : "next release" meaning 1.14, or 1.15?